elektronLogging/logger.go

```diff
@@ -0,0 +1,49 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	. "github.com/spdfg/elektron/elektronLogging/types"
+	"strings"
+	"time"
+)
+
+var config LoggerConfig
+var logger *log.Logger
+var formatter ElektronFormatter
+var ElektronLog *LoggerImpl
+
+func BuildLogger(prefix string) {
+
+	// read configuration from yaml
```

Comment:
Comments should start with capital letters and should end with a full stop.

Reply:
Done